src/svg.resize.js

```diff
@@ -51,22 +50,36 @@ class ResizeHandler {
       return
     }
 
-    // Fire beforedrag event
-    if (this.el.dispatch('beforeresize', { event: e, handler: this }).defaultPrevented) {
-      return
-    }
-
-    this.events.push(e.type)
     this.box = this.el.bbox()
     this.startPoint = this.el.point(startX, startY)
 
-    // We consider the resize done, when a touch is canceled, too
-    const eventMove = (isMouse ? 'mousemove' : 'touchmove') + '.resize'
-    const eventEnd = (isMouse ? 'mouseup' : 'touchcancel.resize touchend') + '.resize'
+    if (e.type === 'rot') { // rotate
+      if (this.el.dispatch('beforerotate', { event: e, handler: this }).defaultPrevented) {
+        return
+      }
+      // We consider the resize done, when a touch is canceled, too
+      const eventMove = (isMouse ? 'mousemove' : 'touchmove') + '.rotate'
+      const eventEnd = (isMouse ? 'mouseup' : 'touchcancel.rotate touchend') + '.rotate'
```

Comment:
There is no reason for replicating this lines here. The only thing different for rotation is the function which is called as event handler. Everything else is teh same.

So do the eventMove = and eventEnd = part once before everything. You also dont need an extra event dispatch for rotation. The event is passed along in details. The user can see what it was